pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -450,7 +450,7 @@ type ObjectStoreSpec struct {
 }
 
 type BucketHealthCheckSpec struct {
-	Bucket        HealthCheckSpec   `json:"rgw,omitempty"`
+	Bucket        HealthCheckSpec   `json:"bucket,omitempty"`
```

Comment:
Should a breaking change entry be added to the pending release notes for this rename?

Reply:
This must be backported, so updating the pending release note won't help much, unfortunately. We will mention this once we release 1.4.2. How does that sound?

Reply:
I see, yeah that should suffice.